### DISCUSSION TITLE
Proxy

### DISCUSSION THREAD

**Discussion by Leks62** [TheR1D/shell_gpt#122] [Q&A] [answered]
How I can use proxy for sgpt?

**Comment by TheR1D** [comment, marked as answer]
You can change `OPENAI_API_HOST` to point to your proxy in runtime config file located in `~/.config/shell_gpt/.sgptrc`.

**Reply by Leks62** [reply]
Does it support socks5 protocol? Сan you give example of OPENAI_API_HOST string?

**Reply by TheR1D** [reply]
https://github.com/TheR1D/shell_gpt/issues/189
